README.md

```diff
@@ -105,6 +105,11 @@ $ make install
 Packetgraph project is published under [GNU GPLv3](http://www.gnu.org/licenses/quick-guide-gplv3.en.html).
 For more information, check LICENSE file.
 
+# Contribute
+
+New to packetgraph ? Want to contribute and/or create a new brick ?
```

Comment:
we should give a link to linux coding style too, as we it: https://www.kernel.org/doc/Documentation/CodingStyle